packages/components/test/Buttons.spec.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react'
+import { renderJSON } from '@theme-ui/test-utils'
+import { ThemeProvider } from 'theme-ui'
+
+import { Button } from '..'
+
+import { theme } from './__test-utils__'
+
+describe('Button', () => {
```

Comment:
Since this describes `Button`, name the file `Button` instead of `Buttons` for consistency?